x/market/genesis.go

```
@@ -48,22 +48,35 @@
 
 	for _, record := range data.Bids {
 		key := keys.MustBidKey(keys.BidStateToPrefix(record.State), record.ID())
+		revKey := keys.MustBidReverseKey(keys.BidStateToPrefix(record.State), record.ID())
 
 		if store.Has(key) {
-			panic(fmt.Errorf("market genesis bids init. bid id %s: %w", record.ID(), types.ErrBidExists))
+			panic(fmt.Errorf("market genesis bids init. bid id %s: %w", record.ID, types.ErrBidExists))
+		}
+		if store.Has(revKey) {
+			panic(fmt.Errorf("market genesis bids init. reverse key for bid id %s: %w", record.ID, types.ErrBidExists))
```

golangci-lint (1.64.8)

SA5009: Printf format %s has arg #1 of wrong type func() github.com/akash-network/akash-api/go/node/market/v1beta4.BidID

(staticcheck)